testsuite/integration/microprofile-tck/rest-client/src/test/java/org/wildfly/cleanup/WiremockCleanup.java

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright The WildFly Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.wildfly.cleanup;
+
+/**
+ * @author <a href="mailto:jperkins@redhat.com">James R. Perkins</a>
+ */
+public class WiremockCleanup {
+
+    public static void main(String[] args) {
+        for (String arg : args) {
+            if (arg.isBlank()) {
+                continue;
+            }
+            System.out.printf("Cleaning up system property: %s=%s%n", arg, System.getProperty("javax.xml.transform.TransformerFactory"));
```

Comment:
I feel lame asking for corrected logging from such an egregious hack, but...

It should be "System.getProperty(arg)"

:-D

Reply:
Doh, yes. Changed.